tox.ini

```diff
@@ -2,9 +2,19 @@
 envlist = py35,py36
 
 [testenv]
+basepython = python3.6
```

Comment:
Could you please elaborate on what happens if there is no python3.6? I am not tox-fluent enough.